app/src/main/java/com/github/se/cyrcle/ui/profile/ViewProfileScreen.kt

```diff
@@ -455,3 +463,126 @@ private fun UserReviewsSection(
+  val showDialogImageDestinationPath = remember { mutableStateOf<List<String>>(emptyList()) }
+  val showDialogImagePath = remember { mutableStateOf<String?>(null) }
+
+  val strResToast = stringResource(R.string.view_profile_screen_image_deleted)
+
+  Box(modifier = Modifier.fillMaxSize()) {
+    if (imagesUrls.isEmpty()) {
+      Text(
+          text = stringResource(R.string.view_profile_screen_no_images_message),
+          style = MaterialTheme.typography.bodyMedium,
+          modifier = Modifier.padding(16.dp).testTag("NoImagesMessage"))
+    } else {
+      LazyColumn(
+          modifier = Modifier.fillMaxWidth().testTag("UserImagesList"),
+          contentPadding = PaddingValues(16.dp)) {
+            itemsIndexed(imagesUrls) { index, url ->
+              UserImageCard(
+                  url = url,
+                  onClick = {
+                    showDialogImage.value = url
+                    showDialogImageDestinationPath.value =
+                        showDialogImageDestinationPath.value.plus(it)
+                    showDialogParking.value =
+                        parkingViewModel.getParkingFromImagePath(imagesPaths[index])
+                    showDialogImagePath.value = imagesPaths[index]
+                  })
+            }
+          }
+    }
+  }
+
+  // Show dialog if an image is clicked
+  showDialogImage.value?.let { imageUrl ->
+    AlertDialog(
+        onDismissRequest = { showDialogImage.value = null },
+        title = { Text(stringResource(R.string.view_profile_screen_image_dialog_title)) },
+        text = {
+          Column {
+            Text(stringResource(R.string.view_profile_screen_image_dialog_description))
+            Spacer(modifier = Modifier.height(16.dp))
+            Image(
+                painter = rememberImagePainter(data = imageUrl),
+                contentDescription = null,
+                modifier = Modifier.fillMaxWidth().height(200.dp))
+          }
+        },
+        confirmButton = {
+          IconButton(
+              modifier = Modifier.padding(10.dp),
+              icon = Icons.Filled.LocalParking,
+              contentDescription = "Go To Parking",
+              testTag = "ParkingFromImageButton",
+              onClick = {
+                parkingViewModel.getParkingById(
+                    showDialogParking.value!!,
+                    { parkingForPath ->
+                      parkingViewModel.selectParking(parkingForPath)
+                      navigationActions.navigateTo(Screen.PARKING_DETAILS)
+                      Toast.makeText(context, strResToast, Toast.LENGTH_SHORT).show()
+                    },
+                    {})
+              })
+        },
+        dismissButton = {
+          IconButton(
+              modifier = Modifier.padding(10.dp),
+              icon =
+                  Icons.Filled
+                      .RestoreFromTrash, // or Icons.Filled.Paid or Icons.Filled.AttachMoney etc...
```

Comment:
Not sure I understand the icon choice here. If we want to delete the image from the cloud, then I don't think that an icon named `RestoreFromTrash` is the best choice, and I was confused testing the UI.
Isn't there a simple `Trash` icon ?

Reply:
There unfortunately isn't...

Reply:
Are you sure? I'm pretty sure it is used in the ZoneManager screen

Reply:
Do you know what it's called then?